decidim-core/lib/decidim/core.rb

```diff
@@ -447,6 +447,11 @@ def self.seed!
     "/"
   end
 
+  # Enable/Disable the service worker
+  config_accessor :service_worker_enabled do
+    true
```

Comment:
I was also thinking could we set the default here based on the environment?
```suggestion
    Rails.env.exclude?("development")
```

This would be nice when we don't use the secrets for the particular environment (e.g. legacy systems that are upgraded).